src/background/background.js

```diff
@@ -2,7 +2,7 @@ import { redirectFromFandom, redirectFromGoogle } from "./redirects.js"
 
 // Any requests beginning with these patterns get intercepted.
 const fandomPattern = "https://pathofexile.fandom.com/wiki/*"
-const googlePattern = "https://*.google.com/search?*q=poe+*"
+const googlePattern = "https://*.google.com/search?*q=poewiki+*"
```

Comment:
```suggestion
const googlePattern = "https://*.google.com/search?*q=poe*wiki+*"
```

May be worth considering matching wildcards to capture spaces or queries like `poe faster attacks wiki` where people might have `wiki` as an afterthought.